config.style

```diff
@@ -1011,12 +1011,28 @@ Layer{
 Layer{
 	id: "SD_SpeciesDistribution"
 	registry-id:""
-	tags: inspire
+	tags: inspire production
```

Comment:
Aren't the production tags obsolete?

Reply:
Sorry, old habit. I will fix it in the next PR I make.